README.md

```diff
@@ -150,12 +150,12 @@ cargo run --bin ln_gateway --release cfg
 ```
 To get an invoice you can use ln2 :
 ```shell
-#make sure your client holds coins for that amount tier. 100000msat = 100sat. For now the client is not able to spend with change
-lightning-cli --network regtest --lightning-dir=ln2 invoice 100000 test test 1m
+#Make sure your peg-in created coins of small enough tiers
+lightning-cli --network regtest --lightning-dir=ln2 invoice 100000 test test 10m
```

Comment:
Uh, does it really take so long sometimes :grimacing: I'd hope not :rofl:

Reply:
When I struggled setting everything up the invoice expiring was extremely annoying (deleting it again or creating one with a different label). That's why I thought it'd be good to give people following the readme more time so this would not be an additional hurdle. (now I wrote a paragraph on something that was probably just a joke xD)

Reply:
Ah right, I was still in the integration test mindset where everything is run consecutively. Forgot that humans require more time. Damn wetware … :laughing: